.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "editor.formatOnSave": true
```

Comment:
Should we commit vscode user settings ?

Reply:
No, we should not. Please @drillinP  ignore this file by editing `.git/info/exclude` and adding this file.

Reply:
@nboisteault @drillinP I'll do it

Reply:
Why `.git/info/exclude` and not just adding it in the `.gitignore`  in the repository? (the `.vscode` directory) ?

Reply:
Yes we can do it this way but in this case we'll have to handle every files/directories created by different IDE, tools, etc.
As vscode is very popular, let's add this directory. It will avoid this directory to be committed by potential contributors in future.

Reply:
Done in `.gitignore`